### ISSUE TITLE
allow the selection of XLS in runner

### ISSUE THREAD

**Issue by bmatraves** [a85/POSTMan-Chrome-Extension#6739]
**Is your feature request related to a problem? Please describe.**
If you collect data in a spreadsheet, it is quite tedious to break tabs in to individual CSV files for use in runner (well, its not the end of the world, but it could be slicker)

**Describe the solution you'd like**
In runner, for data "select file" allow also common spreadsheet formats, such as XLS, XLSX and allow the selection of TAB to identify the data sheet.

**Describe alternatives you've considered**
Saving XLS tabs in to separate CSV is the only option manually or by using python prior to using runner

**Additional context**
The problem of saving individual tabs to CSV is the data becomes un-linked from the source, so you have to do this as many times as you edit the source spreadsheet.  it would be simpler to reference one source file and have runner break it down into CSV sheets, use CSV and then delete CSV after run.

**Comment by shamasis** [comment]
Yeah. It would definitely be slicker. Would need to explore how difficult it is to actively support all quirks of different excel versions and so many third party excel compatible file formats. 

Maybe this in future could become part of the plugins concepts we are working on so that our community can give it more love than we can. I’m keeping this as a feature request for now. 

PS: we recently fixed a lot of CSV issues in our v7.2.2 of app. Do check that out in case you were facing any bugs earlier.

**Comment by ksultz** [comment]
Another vote for this enhancement please.  Other API test tools have this capability, so it would be good for Postman to have it.
